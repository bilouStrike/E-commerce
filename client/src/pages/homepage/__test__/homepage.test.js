import { shallow } from 'enzyme';
import React from 'react';
import HomePage from '../homepage.component';
import toJson from 'enzyme-to-json';

it('expect to render home page', () =>{
    expect(toJson(shallow(<HomePage/>))).toMatchSnapshot();
});