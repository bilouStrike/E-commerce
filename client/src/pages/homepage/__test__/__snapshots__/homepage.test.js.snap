// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render home page 1`] = `
<styled.div>
  <h3>
    Welcome to my Homepage
  </h3>
  <Connect(Directory) />
</styled.div>
`;
